packages/cpdf-source/cpdf-source.2.6/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: "contact@coherentgraphics.co.uk"
+license: "Coherent Graphics Ltd Non-Commercial Use License Agreement"
+build: [[make]]
+depends: [
+  "ocaml" {>= "4.10.0"}
+  "ocamlfind" {build}
+  "camlpdf" {= version}
+]
+homepage: "http://github.com/johnwhitington/cpdf-source"
+authors: ["John Whitington"]
+bug-reports: "http://github.com/johnwhitington/cpdf-source/issues"
+dev-repo: "git+https://github.com/johnwhitington/cpdf-source"
+install: [[make "install"]]
+synopsis: "High-level PDF tools based on CamlPDF"
+extra-files: ["cpdf.install" "md5=aca30b2d4fec0e5a38374cfefee70a33"]
```

Comment:
btw, i'm wondering, why is this an extra-file if it's already in the source? You would just need to move this file from `files/` to the root directory of your project and there wouldn't be any need for this extra-file

Reply:
Thanks. Previously, I didn't use `opam-publish`, and so I didn't include the `opam` file or the `cpdf.install` file in the source at all.

If opam-publish/opam know what to do with a file in the root of the source called `<packagename>.install` automatically, then I think I need to:

1) move files/cpdf.install to cpdf.install
2) remove the extra-files entry from opam
3) rewrite the v2.6 tag to point to the tip of trunk so these new files are included
4) re-run opam-publish

Does that sound about right?

Reply:
> rewrite the v2.6 tag to point to the tip of trunk so these new files are included

your project seems to be packaged by other package managers and 2.6 seems to already be in for some of them. I would recommend not to do this step in case that breaks them and create a new `v2.6-1` tag instead.

We can also simply add `mv files/<pkgname>.install .` to the `build` rules. I think it's much easier and quicker. You can move the .install in your main branch instead so that we won't have to add this rule next time

Reply:
Won't `v2.6-1` interfere with `"camlpdf" {= version}` in the opam file, or does anything after the `-` not count?

Reply:
you're correct i missed that. Fair enough. Let's change this here PR instead. I'll do don't worry